### DISCUSSION TITLE
French Translation

### DISCUSSION THREAD

**Discussion by lilrawry** [goatshriek/stumpless#332] [Q&A] [answered]
Can i Translate your project in FR

**Comment by goatshriek** [comment]
We certainly welcome any and all translation support! There is already a French translation, but it is missing some strings that have been added since it was done (marked with a `todo translate` comment). If you submit a PR with these strings translated, it will be merged in pretty quickly.

If you want to add more complete French documentation, you can add a new `fr-fr` folder for it within the project `l10n` folder. For this, just provide enough translation of the README file to let a newcomer know what the project is, you don't need to do the entire thing. You would be the first to make one of these types of contributions, so there isn't heavy documentation on this path just yet - right now this folder only has translations of the code of conduct.

**Reply by lilrawry** [reply]
alrighty thanks for your reply, i'll translate it soon as possible, could you please send me the file link cause i'm lost finding the `french-translation` thanks 
best regards!

**Reply by goatshriek** [reply, marked as answer]
The current french translation is at [include/private/config/locale/fr-fr.h](https://github.com/goatshriek/stumpless/blob/latest/include/private/config/locale/fr-fr.h).

**Reply by lilrawry** [reply]
> The current french translation is at [include/private/config/locale/fr-fr.h](https://github.com/goatshriek/stumpless/blob/latest/include/private/config/locale/fr-fr.h).

copy that, done ✅.
